BananaCrystal/bananacrystal-payment-gateway
# Fix subscriber params and verify subscriptions payload

## What

* Rename subscriber user id param
* Encode base64 the subscriber params/attributes when redirecting to pay
* Verify Payload for IPN notification with a subscription payload

## Why

* User ID and Username are passed in the params
* Payload needs to be verified before using the data in the payload to create the subscription

## How to test

1. Checkout with a subscription
2. Verify that the subscription got paid

## References/Links
